warmup.py

```diff
@@ -240,17 +254,36 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        points = np.atleast_2d(points)
+        for i in range(len(points[:-1])):
+            for j in range(len(points[i+1:])):
+                dist_vect = points[j+i+1] - points[i]
+                dist_vect -= L*np.round(dist_vect/L) 
+                dist_mag = np.linalg.norm(dist_vect)
+                if self.rmin <= dist_mag <= self.rmax:
+                    bin_coord = int(np.floor(dist_mag/self.dr))
```

Comment:
This won't work right if `self.rmin` is not zero. Can you account for this shift in the beginning of the bin range?